src/frontend-pwa/src/utils/SendAnalytics.ts

```diff
@@ -9,7 +9,8 @@ import axios from 'axios';
 import constants from '../constants/Constants';
 import Analytic from '../Type/Analytic';
 
-const Analytics = (request: Analytic) => {
+// TODO remove console.logs
```

Comment:
Is this console log staying for this PR?

Reply:
@BradyMitch haha yeah, Matthew asked the same thing. I want it in there for my next ticket